src/media/Router.ts

```diff
@@ -143,6 +147,66 @@ export class Router extends EventEmitter {
 		return canConsume;
 	}
 
+	@skipIfClosed
+	public async createActiveSpeakerObserver({
```

Comment:
I think we need to call this method (in constructor?) or middleware will never handle the messages from media-node, since there is no `ActiveSpeakerObserver` on the router.